packages/components/src/components/PipelineRuns/PipelineRuns.js

```diff
@@ -141,27 +147,21 @@ const PipelineRuns = ({
 
     return {
       id: `${namespace}:${pipelineRunName}`,
-      name: url ? (
-        <Link to={url} title={pipelineRunName}>
+      name: pipelineRunURL ? (
+        <Link to={pipelineRunURL} title={pipelineRunName}>
           {pipelineRunName}
         </Link>
       ) : (
         pipelineRunName
       ),
       pipeline:
-        !hidePipeline &&
-        (pipelineRefName ? (
-          <Link
-            to={createPipelineRunsByPipelineURL({
-              namespace,
-              pipelineName: pipelineRefName
-            })}
-            title={pipelineRefName}
-          >
+        pipelineRefName &&
+        (pipelineRunsByPipelineURL ? (
+          <Link to={pipelineRunsByPipelineURL} title={pipelineRefName}>
             {pipelineRefName}
           </Link>
         ) : (
-          ''
+          pipelineRefName
```

Comment:
Will this cause bad rendering for inline pipeline runs ?

Reply:
`pipelineRefName` will be falsey in that case so nothing gets rendered as expected
example included in the storybook with no pipelineRef